doc/api/addons.markdown

```diff
@@ -868,9 +978,13 @@ Test in JavaScript by running:
     // test.js
     const addon = require('./build/Release/addon');
 
-[online]: https://v8docs.nodesource.com/
+[online]: https://V8docs.nodesource.com/
 [libuv]: https://github.com/libuv/libuv
 [download]: https://github.com/nodejs/node-addon-examples
 [node-gyp]: https://github.com/nodejs/node-gyp
-[v8 reference]: http://izs.me/v8-docs/main.html
-[Embedder's Guide]: https://code.google.com/apis/v8/embed.html
+[V8 reference]: http://izs.me/V8-docs/main.html
+[Embedder's Guide]: https://code.google.com/apis/V8/embed.html
```

Comment:
This link gives a `404` - looks like they moved it to https://developers.google.com/v8/intro or for the embedder's guide https://developers.google.com/v8/embed